README.md

```diff
@@ -328,6 +328,16 @@ external dependencies. But running `riff run cargo build` succeeds because Riff
 is able to infer which external dependencies are missing and provide them in the
 background using Nix.
 
+## Garbage Collection
+
+While using [Nix] grants Riff its powers, it also comes with some drawbacks,
+one of which being the issue of disk space. After using Riff on a few projects,
+you may notice the `/nix/store` directory grow in size. Luckily, it is
+relatively easy to reclaim this space by running `nix-collect-garbage`.
```

Comment:
I think we should use the unified CLI variant here. AFAIK `nix store gc` is the equivalent.

Reply:
The only reason I didn't was because of https://github.com/NixOS/nix/issues/7063 -- `nix store gc` doesn't do anything to communicate to the user that garbage is indeed being collected unless you pass it `-v`, while `nix-collect-garbage` does communicate progress by default.

I suppose I could suggest `nix store gc -v` though...

Reply:
An entirely fair point and good to know. I'd say leave it as-is for now and we can revisit.